src/main/sensors/boardalignment.c

```diff
@@ -59,28 +61,59 @@ void initBoardAlignment(const boardAlignment_t *boardAlignment)
     rotationAngles.angles.pitch = degreesToRadians(boardAlignment->pitchDegrees);
     rotationAngles.angles.yaw   = degreesToRadians(boardAlignment->yawDegrees  );
 
-    buildRotationMatrix(&rotationAngles, boardRotation);
+    buildRotationMatrix(&rotationAngles, &boardRotation);
 }
 
-static void alignBoard(float *vec)
+FAST_CODE static void applyRotation(float *vec, fp_rotationMatrix_t *rotationMatrix)
```

Comment:
This code is duplicated. It should be unified in some math library.

Reply:
yeah, I thought it might be duplicated when the method was refactored from using `boardRotation` to any rotation matrix...  I will investigate, however this one is marked as FAST_CODE which may or may not complicate things, library wise.

Reply:
done